src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  comment: {
+    name,
+    email,
+    body,
+  },
+}) => {
+  return (
```

Comment:
You can remove return statement from here if you use round braces instead of curly